include/mrdocs/Generator.hpp

```diff
@@ -64,6 +64,7 @@ class MRDOCS_VISIBLE
         @li "adoc" Asciidoctor
         @li "xml" XML
         @li "html" HTML
+        @li "tagfile" Tagfile
```

Comment:
Shouldn't the tagfile generator be associated with a generator? Different generators with different options would generate different tagfiles and there's always a 1-to-1 relationship. 

For instance, at the very least, https://github.com/boostorg/url/blob/develop/doc/tagfiles/boost-url-doxygen.tag.xml uses .adoc for the file extensions. And any option that influences what files are generated should be reflected in the tagfile. So some generators, tagfiles don't make much sense.

It also seems easier to implement.